portingdb/history_graph.py

```diff
@@ -0,0 +1,95 @@
+from collections import defaultdict
+
+from plotly.offline import plot
+from plotly.offline.offline import get_plotlyjs
+from plotly.graph_objs import Scatter, Figure, Layout
+
+STATUS_ORDER = (
+    'dropped',
+    'py3-only',
+    'legacy-leaf',
+    'released',
+    'in-progress',
+    'mispackaged',
+    'idle',
+    'blocked',
+)
+
+def order_key(status_ident):
+    try:
+        return STATUS_ORDER.index(status_ident)
+    except ValueError:
+        return 4
```

Comment:
What means 4?:)

Reply:
It puts any unknown status somewhere between "ported" and "unported". I left that grossly underdocumented, sorry! (but the actual value not that important)